Card/src/main/java/com/braintreepayments/api/card/BinData.kt

```diff
@@ -0,0 +1,147 @@
+package com.braintreepayments.api.card
+
+import android.os.Parcelable
+import androidx.annotation.RestrictTo
+import androidx.annotation.StringDef
+import com.braintreepayments.api.sharedutils.Json
+import kotlinx.parcelize.Parcelize
+import org.json.JSONObject
+
+/**
+ * A class to contain BIN data for the card number
+ */
+@Parcelize
+data class BinData(
+    private var prepaid: String = UNKNOWN,
+    private var healthcare: String = UNKNOWN,
+    private var debit: String = UNKNOWN,
+    private var durbinRegulated: String = UNKNOWN,
+    private var commercial: String = UNKNOWN,
+    private var payroll: String = UNKNOWN,
+    private var issuingBank: String = UNKNOWN,
+    private var countryOfIssuance: String = UNKNOWN,
+    private var productId: String = UNKNOWN
+) : Parcelable {
+
+    @Retention(AnnotationRetention.SOURCE)
+    @StringDef(YES, NO, UNKNOWN)
+    internal annotation class BinType
+
+    /**
+     * @return Whether the card is a prepaid card. Possible [BinType] values are [BinData.YES], [BinData.NO] or [BinData.UNKNOWN]
+     */
+    @BinType
+    fun getPrepaid(): String {
+        return prepaid
+    }
```

Comment:
random thought: We may be able to remove these accessors since it's now a Kotlin `data class`. We can migrate the docs strings into `@property` annotations as well.